chipy_org/apps/job_board/management/commands/post_approved_jobs_to_slack.py

```diff
@@ -0,0 +1,54 @@
+import json
+import logging
+import datetime
+
+import requests
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from django.template import loader
+
+from chipy_org.apps.job_board.models import JobPost
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    help = """
+    This command retrieves all approved and active job posts from the database
+    and posts them to the slack jobs channel. 
+    
+    Integrate this command with a scheduler to automatically post jobs to the 
+    slack jobs channel.  
+    """
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "weekdays",
+            nargs="+",
+            type=int,
+            help="""
+            The desired weekdays to post represented an integer where Monday=0, Tuesday=1, ... , Saturday=5, and Sunday=6.
+            Posts to the slack channel will only occur on the selected weekdays.
+            """,
+        )
+
+    def handle(self, *args, **options):
+        posts = JobPost.approved_and_active.all()
+
+        if posts.count() and datetime.date.today().weekday() in options["weekdays"]:
```

Comment:
Truly fine in the context of adding the code, but from a style perspective this is a little muddied with the concerns of the command. Ideally, I think we'd be well-served to put most of this into a Slack facade that handles renders and requests calls.

Reply:
That's a reasonable idea and an opportunity for me to practice a design pattern.  Though I think I'll leave it until we add slack use case number two if that's cool.